pom.xml

```diff
@@ -96,7 +96,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
     <failIfNoTests>false</failIfNoTests>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-    <jetty.version>9.4.35.v20201120</jetty.version>
+    <jetty.version>9.4.41.v20210516</jetty.version>
```

Comment:
Any reason for not using ``9.4.43.v20210629``? Or we just want to cover the CVE fix?

Reply:
We may want to go with 9.4.43, as the most recent CVE is only fixed in that version: https://github.com/eclipse/jetty.project/security/advisories/GHSA-vjv5-gp2w-65vm